pytest/tests/stress/hundred_nodes/analyze/block_by.py

```diff
@@ -0,0 +1,14 @@
+node_blocks = [0] * 100
+block_by = {}
+for i in range(100):
+    with open(f'pytest-node-{i}.txt') as f:
```

Comment:
can we clean up this a bit? It looks outdated.

Reply:
Sure. The problem is I don't exactly remember which was outdated 😂 but I guess you and Marcelo has already keep one or two useful, up to date scripts, So I can simply delete all `analyze` scripts here?

Reply:
That's also fine I guess

Reply:
Deleted. If you feel some of your analyze script help, please add to this branch